src/client/components/equalizer/equalizer.js

```diff
@@ -172,9 +206,9 @@ class Equalizer extends Component {
     ctx.fill();
   }
 
-  setCanvasToState = (canvasEl) => {
-    this.setState({
-      ctx: canvasEl.getContext('2d'),
+  setCanvasToState = async (canvasEl) => {
```

Comment:
Why this function needed to be asynchronous?

Reply:
This function should not be asynchronous
fixed it